inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/internal/JdbcMultiBatchingOutputFormat.java

```diff
@@ -519,6 +512,9 @@ protected void attemptFlush() throws IOException {
                     if (!flushFlag && null != tableException) {
                         LOG.info("Put tableIdentifier:{} exception:{}",
                                 tableIdentifier, tableException.getMessage());
+                        outputMetrics(tableIdentifier, Long.valueOf(tableIdRecordList.size()),
+                                1L, true);
+                        dirtySinkHelper.invoke(record, DirtyType.RETRY_LOAD_ERROR, tableException);
                         tableExceptionMap.put(tableIdentifier, tableException);
                         if (isIgnoreTableException) {
```

Comment:
It is better to rethrow the exception when 'isIgnoreTableException' is 'false'?

Reply:
has log maxRetries times before